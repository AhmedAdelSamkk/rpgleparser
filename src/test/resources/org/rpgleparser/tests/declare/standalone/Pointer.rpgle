        DCL-S pUserspace POINTER;
        DCL-S pProc POINTER(*PROC);