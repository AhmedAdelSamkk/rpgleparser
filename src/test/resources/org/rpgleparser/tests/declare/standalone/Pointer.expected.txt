/*===TOKENS===*/
DS_Standalone            <DCL-S>
ID                       <pUserspace>
ID                       <POINTER>
FREE_SEMI                <;>
DS_Standalone            <DCL-S>
ID                       <pProc>
ID                       <POINTER>
'('                      <(>
ID                       <*PROC>
')'                      <)>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (dspec
    DCL-S
    (identifier (free_identifier pUserspace))
    (datatype (identifier (free_identifier POINTER)))
    ;
  )
  (dspec
    DCL-S
    (identifier (free_identifier pProc))
    (datatype
      (identifier (free_identifier POINTER))
      (args ( (expression (identifier (free_identifier *PROC))) ))
    )
    ;
  )
)
/*======*/